docker/setup_configuration/data.yaml

```diff
@@ -1,5 +1,5 @@
-openklant_tokens_config_enable: true
-openklant_tokens:
+zgw_tokens_tokenauth_config_enable: true
+zgw_tokens_tokenauth:
```

Comment:
I thought we mentioned to call this `tokenauth_config_enable`? This prefix doesn't apply for Objects/Objecttypes, because those are not part of Zaakgericht werken

Reply:
@danielmursa-dev and I discussed that adding a prefix for the library that will be responsible for defining these tokens would be nice, because it takes away some work in advance as the configuration files will have the correct namespaces already, hence the `zgw_tokens` prefix. I guess the question here is, how are we going to call the library and use that prefix?

Reply:
Hmm okay, I'll approve for now so that we can get the other PR in Objecttypes merged as well. We'll probably have to take a look at all the namespaces next week anyway

Reply:
I see that in https://github.com/maykinmedia/objecttypes-api/pull/135/files this was changed to just `tokenauth`, could you change it to that as well? I think no prefix is better than a prefix that might be confusing